Azure.ResourceManager.Core/SubscriptionContainer.cs

```diff
@@ -12,15 +12,13 @@ namespace Azure.ResourceManager.Core
     /// </summary>
     public class SubscriptionContainer : OperationsBase
     {
-        public static readonly ResourceType AzureResourceType = "Microsoft.Resources/subscriptions";
+        public static readonly ResourceType ResourceType = "Microsoft.Resources/subscriptions";
```

Comment:
I think we should remove these from the container classes and only have it on the operations class

Reply:
Agree, it a duplicate definition. Will update soon.